scalameta/dialects/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -173,6 +181,7 @@ package object dialects {
     def allowOrTypes = Scala211.allowOrTypes
     def allowTraitParameters = Scala211.allowTraitParameters
     def allowLiteralTypes: Boolean = false // Scheduled to be included in 2.12.2
+    def allowTrailingComma: Boolean = true // Schedule to be included in 2.12.2
```

Comment:
both allowTrailingComma and allowLiteralTypes and scheduled for 2.12.2, but one is false and the other is true?